client/src/components/app/app.jsx

```diff
@@ -7,6 +7,8 @@ import {findAvgRating, sortByCriteria} from '../../../../helper/reviewsHelper.js
 import ReviewsErrorBoundary from '../errorBoundary/reviewsErrorBoundary.jsx';
 import OverviewErrorBoundary from '../errorBoundary/overviewErrorBoundary.jsx'
 
+export const BACKEND_URL = process.env.NODE_ENV === 'development' ? `http://localhost:3000` : `http://34.225.154.204`;
```

Comment:
After you merge the PR and pull it down to your org's local repo, when you try to pull from org's main branch to your forked copy's local main, you'll have to change the IP Address.